nominatim/tools/special_phrases.py

```diff
@@ -0,0 +1,249 @@
+"""
+    Functions to import special phrases into the database.
+"""
+import logging
+import os
+import re
+import subprocess
+import sys
+import json
+from os.path import isfile
+from icu import Transliterator # pylint: disable-msg=no-name-in-module
```

Comment:
icu is supposed to be the PyICU python module, to be honest I didn't get why I got this pylint error that's why I ignored it.

Reply:
Ok you were right since PyICU is generating the object dynamically to wrap the C++ one, it wasn't detected by pylint. Adding icu to extension-pkg-whitelist doesn't solve the problem, some people reported this on pylint's github, so I went for the solution with ignored-modules=icu instead.